Use OGRE_PLUGIN_DIR as variable because it can be set from debian/rules.
--- a/src/config.h.in
+++ b/src/config.h.in
@@ -17,7 +17,7 @@
 #   define SUMWARS_OGRE_PLUGIN_DIR "@SUMWARS_OGRE_PLUGIN_DIR_DBG@"
 #	define SUMWARS_DEBUG_POSTFIX "@CMAKE_DEBUG_POSTFIX@"
 #else
-#   define SUMWARS_OGRE_PLUGIN_DIR "@SUMWARS_OGRE_PLUGIN_DIR_REL@"
+#   define SUMWARS_OGRE_PLUGIN_DIR "@SUMWARS_OGRE_PLUGIN_DIR@"
 #	define SUMWARS_DEBUG_POSTFIX ""
 #endif
 
